### PR TITLE
Cleanup repo layout

### PR DESCRIPTION
**Pull request by decriptor**

### CODE REVIEW

**Comment by trixtur** [comment]
Tried to run the debugger on the executable. I couldn't get the ConfigurationManager to read the config file anymore. Where does it need to go?

**Review by trixtur — CHANGES_REQUESTED** [review]
We need to be able to pull the configuration from the web service.

File: src/CsBot/IrcBotService.cs [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+			else
+				settingsJson = LoadLocalIrcConfig (Path.Combine (Directory.GetCurrentDirectory (), _settingsFileLocation));
+
+			Settings = JsonConvert.DeserializeObject<IrcServerOptions> (settingsJson);
+
+			_log.LogInformation ("Settings Loaded");
+		}
+
+		async Task<string> DownloadIrcConfig (Uri uri)
+		{
```
Comment (decriptor) [inline src/CsBot/IrcBotService.cs:309]: @trixtur Here is the code to download a configuration file. `WebClient` is dated and is replaced by httpclient

**Review by trixtur — APPROVED** [review]
Loaded it up and ran some commands. Looks good!

**Pull request merged**